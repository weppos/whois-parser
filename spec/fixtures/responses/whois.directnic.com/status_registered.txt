
Domain Name: DIRECTNIC.COM
Registry Domain ID: 19525357_DOMAIN_COM-VRSN
Registrar WHOIS Server: whois.directnic.com
Registrar URL: http://www.directnic.com
Updated Date: 2016-08-09T12:54:08Z
Creation Date: 2000-02-09T17:22:12Z
Registrar Registration Expiration Date: 2020-02-09T17:22:12Z
Registrar: DNC Holdings, Inc.
Sponsoring Registrar IANA ID: 291
Registrar Abuse Contact Email: abuse@directnic.com
Registrar Abuse Contact Phone: +1.8778569598
Domain Status: clientDeleteProhibited (https://www.icann.org/epp#clientDeleteProhibited)
Domain Status: clientTransferProhibited (https://www.icann.org/epp#clientTransferProhibited)
Domain Status: clientUpdateProhibited (https://www.icann.org/epp#clientUpdateProhibited)
Registrant Name: Direct Privacy
Registrant Organization: Savvy Investments, LLC Privacy ID# 472417
Registrant Street: 2123 Pioneer Ave
Registrant City: Cheyenne
Registrant State/Province: WY
Registrant Postal Code: 82001
Registrant Country: US
Registrant Phone: +1.5043215074
Registrant Phone Ext:
Registrant Fax:
Registrant Fax Ext:
Registrant Email: directnic.com@domainvault.io
Admin Name: Direct Privacy
Admin Organization: Savvy Investments, LLC Privacy ID# 472417
Admin Street: 2123 Pioneer Ave
Admin City: Cheyenne
Admin State/Province: WY
Admin Postal Code: 82001
Admin Country: US
Admin Phone: +1.5043215074
Admin Phone Ext:
Admin Fax:
Admin Fax Ext:
Admin Email: directnic.com@domainvault.io
Tech Name: Direct Privacy
Tech Organization: Savvy Investments, LLC Privacy ID# 472417
Tech Street: 2123 Pioneer Ave
Tech City: Cheyenne
Tech State/Province: WY
Tech Postal Code: 82001
Tech Country: US
Tech Phone: +1.5043215074
Tech Phone Ext:
Tech Fax:
Tech Fax Ext:
Tech Email: directnic.com@domainvault.io
Name Server: NS0.DIRECTNIC.COM
Name Server: NS1.DIRECTNIC.COM
URL of the ICANN WHOIS Data Problem Reporting System
http://wdprs.internic.net
>>> Last update of WHOIS database: 2018-05-03T20:26:18Z <<<
For more information on Whois status codes,
please visit https://www.icann.org/resources/pages/epp-status-codes-2014-06-16-en.


The compilation, repackaging, dissemination, or other use of this WHOIS
data is expressly prohibited without the prior written consent of
DNC Holdings, Inc.

DNC Holdings reserves the right to terminate your access to its WHOIS
database in its sole discretion, including without limitation, for
excessive querying of the database or for failure to otherwise abide by
this policy.

DNC Holdings reserves the right to modify these terms at any time.

NOTE: THE WHOIS DATABASE IS A CONTACT DATABASE ONLY.
LACK OF A DOMAIN RECORD DOES NOT SIGNIFY DOMAIN AVAILABILITY.
