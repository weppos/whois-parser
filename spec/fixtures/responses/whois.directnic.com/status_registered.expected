#domain
  %s == "corehub.net"

#domain_id
  %s == "19525357_DOMAIN_COM-VRSN"

#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true

#created_on
  %s %CLASS{time}
  %s %TIME{2000-02-09 17:22:12 UTC}

#expires_on
  %s %CLASS{time}
  %s %TIME{2020-02-09 17:22:12 UTC}

#registrar
  %s %CLASS{registrar}
  %s.id           == "291"
  %s.name         == "DNC Holdings, Inc."
  %s.organization == "DNC Holdings, Inc."
  %s.url          == "http://www.directnic.com"

#registrant_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_REGISTRANT
  %s[0].name          == "CORE ASSOCIATION"
  %s[0].organization  == ""
  %s[0].address       == ""
  %s[0].city          == "Geneva"
  %s[0].zip           == "CH-1215"
  %s[0].state         == "WY"
  %s[0].country_code  == "US"
  %s[0].phone         == "+41.229295744"
  %s[0].fax           == "+41.229295745"
  %s[0].email         == "directnic.com@domainvault.io"

#admin_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_ADMINISTRATIVE
  %s[0].name          == "Direct Privacy"
  %s[0].organization  == "Savvy Investments, LLC Privacy ID# 472417"
  %s[0].address       == "2123 Pioneer Ave"
  %s[0].city          == "Cheyenne"
  %s[0].zip           == "82001"
  %s[0].state         == "WY"
  %s[0].country_code  == "US"
  %s[0].phone         == "+1.5043215074"
  %s[0].fax           == ""
  %s[0].email         == "directnic.com@domainvault.io"

#technical_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_TECHNICAL
  %s[0].name          == "Direct Privacy"
  %s[0].organization  == "Savvy Investments, LLC Privacy ID# 472417"
  %s[0].address       == "2123 Pioneer Ave"
  %s[0].city          == "Cheyenne"
  %s[0].zip           == "82001"
  %s[0].state         == "WY"
  %s[0].country_code  == "US"
  %s[0].phone         == "+1.5043215074"
  %s[0].fax           == ""
  %s[0].email         == "directnic.com@domainvault.io"

#nameservers
  %s %CLASS{array}
  %s %SIZE{2}
  %s[0] %CLASS{nameserver}
  %s[0].name == "ns0.directnic.com"
  %s[1] %CLASS{nameserver}
  %s[1].name == "ns1.directnic.com"
